######################## [START LOG] ########################
ClassicUO [STANDARD_BUILD] - 3.23.1.0 - 08/24/2025 17:02:20
OS: Win32NT x64
Thread: CUO_MAIN_THREAD

Shard: 74.91.120.104
ClientVersion: 7.0.15.1

Exception:
System.AggregateException: Se han producido uno o varios errores. ---> System.OverflowException: Las dimensiones de la matriz superaron el intervalo admitido.
   en ClassicUO.Assets.TileDataLoader.<Load>b__10_0() en D:\workspace\2024\TazUO\src\ClassicUO.Assets\TileDataLoader.cs:línea 86
   en System.Threading.Tasks.Task.Execute()
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   en System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   en System.Threading.Tasks.Task.Wait(TimeSpan timeout)
   en ClassicUO.Assets.UOFileManager.Load(ClientVersion version, String basePath, Boolean useVerdata, String lang) en D:\workspace\2024\TazUO\src\ClassicUO.Assets\UOFileManager.cs:línea 122
   en ClassicUO.Client.Load() en D:\workspace\2024\TazUO\src\ClassicUO.Client\Client.cs:línea 188
   en ClassicUO.Client.Run() en D:\workspace\2024\TazUO\src\ClassicUO.Client\Client.cs:línea 66
   en ClassicUO.Bootstrap.Main(String[] args) en D:\workspace\2024\TazUO\src\ClassicUO.Client\Main.cs:línea 286
---> (Nº de excepción interna 0) System.OverflowException: Las dimensiones de la matriz superaron el intervalo admitido.
   en ClassicUO.Assets.TileDataLoader.<Load>b__10_0() en D:\workspace\2024\TazUO\src\ClassicUO.Assets\TileDataLoader.cs:línea 86
   en System.Threading.Tasks.Task.Execute()<---

######################## [END LOG] #####################
######################## [START LOG] ########################
ClassicUO [STANDARD_BUILD] - 3.23.1.0 - 08/24/2025 17:02:20
OS: Win32NT x64
Thread: CUO_MAIN_THREAD

Shard: 74.91.120.104
ClientVersion: 7.0.15.1

Exception:
System.AggregateException: Se han producido uno o varios errores. ---> System.OverflowException: Las dimensiones de la matriz superaron el intervalo admitido.
   en ClassicUO.Assets.TileDataLoader.<Load>b__10_0() en D:\workspace\2024\TazUO\src\ClassicUO.Assets\TileDataLoader.cs:línea 86
   en System.Threading.Tasks.Task.Execute()
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   en System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   en System.Threading.Tasks.Task.Wait(TimeSpan timeout)
   en ClassicUO.Assets.UOFileManager.Load(ClientVersion version, String basePath, Boolean useVerdata, String lang) en D:\workspace\2024\TazUO\src\ClassicUO.Assets\UOFileManager.cs:línea 122
   en ClassicUO.Client.Load() en D:\workspace\2024\TazUO\src\ClassicUO.Client\Client.cs:línea 188
   en ClassicUO.Client.Run() en D:\workspace\2024\TazUO\src\ClassicUO.Client\Client.cs:línea 66
   en ClassicUO.Bootstrap.Main(String[] args) en D:\workspace\2024\TazUO\src\ClassicUO.Client\Main.cs:línea 286
---> (Nº de excepción interna 0) System.OverflowException: Las dimensiones de la matriz superaron el intervalo admitido.
   en ClassicUO.Assets.TileDataLoader.<Load>b__10_0() en D:\workspace\2024\TazUO\src\ClassicUO.Assets\TileDataLoader.cs:línea 86
   en System.Threading.Tasks.Task.Execute()<---

######################## [END LOG] #####################
